### DISCUSSION TITLE
Selective hide/show badges?

### DISCUSSION THREAD

**Discussion by jtakalai** [community/community#34540] [Other Feature Feedback, Questions, & Ideas]
Some of the badges could be considered less... prestigious. Yes, I merged a branch without waiting for reviews, does this mean I need to be firebranded with a YOLO achievement?  :D   This one is harmless, to me, but still: these achievements aren't necessary positive in all circumstances to all people, and showing them in the public profile non-consensually feel just wrong. Achievement badges should be opt-in to show, not opt-out, and *definitely* not mandatory.

TL;DR: consider
* not showing achievement badges at all by default
* adding them as an option to enable in the profile settings page

**Comment by jfullstackdev** [comment]
yes you have the option to disable that, even make your profile private, the only thing that's not available is to select what badges you want to be displayed

https://github.com/settings/profile

![image](https://user-images.githubusercontent.com/47092464/193062535-a2a7d6cf-ede5-40f7-9c17-2805ae1ca6e5.png)

**Comment by bever1337** [comment]
It's an incredibly unprofessional sounding achievement. Sometimes it's reasonable to merge without a review, sometimes that would get a developer fired. This is an opt-out system, so developers may have no reason to opt-out until GitHub gives them a tactless achievement. 

My request: GitHub achievements should only cast developers in a positive light.

**Reply by kospl** [reply]
I bet single developers don't get (meaning understand) this badge at all.

**Comment by bever1337** [comment]
Related discussion here: https://github.com/orgs/community/discussions/18542

**Comment by wshanks** [comment]
Selecitve hide/show would be nice. The arctic code vault and Mars rover badges were kind of cool, but I don't want to have a whole array of somewhat frivolous badges displayed like closing an issue within five minutes or having a PR merged.

**Reply by wshanks** [reply]
Oh, I just discovered it's possible by clicking on the badges individually in `https://github.com/<username>?tab=achievements` and selecting "hide from profile"

**Reply by radar-blip** [reply]
Right 
![hideit](https://user-images.githubusercontent.com/39196767/204676067-5636f131-3950-4db3-8d46-f1ba92b5f071.png)

**Reply by therealdhrxv** [reply]
Thank you so much @wshanks 👍👍

**Reply by nikzaddev99** [reply]
how to make it unhide

**Reply by nikzaddev99** [reply]
?

**Reply by radar-blip** [reply]
If you have any achievements they would show up here: https://github.com/nikzaddev99?tab=achievements

**Reply by nikzaddev99** [reply]
that was it

**Comment by jfullstackdev** [comment]
a bump on this one, yup as for now, this can be hidden I tried hiding the YOLO badge as requested

![image](https://user-images.githubusercontent.com/47092464/204708466-d9a09e16-3bb0-4d05-b296-fa3bd99139a3.png)

go to Tab Achievements, click any badge and as demonstrated above, you can now hide it, and it's working as expected

@jtakalai @wshanks
